packages/core/README.md

```diff
@@ -253,48 +313,102 @@ Whitelisted and confirmed parameters include:
     "version": "1.3.0",
     "transparent": true,
     "layers": ["A", "B"]
+  },
+  {
+    "id": "my-self-defined-wmts",
+    "urls": [
+      "url1/{TileMatrix}/{TileCol}/{TileRow}.png",
+      "url2/{TileMatrix}/{TileCol}/{TileRow}.png",
+      "url3/{TileMatrix}/{TileCol}/{TileRow}.png"
+    ],
+    "typ": "WMTS",
+    "format": "image/png",
+    "coordinateSystem": "EPSG:3857",
+    "origin": [-20037508.3428, 20037508.3428],
+    "transparent": false,
+    "tileSize": "256",
+    "minScale": "1",
+    "maxScale": "2500000",
+    "tileMatrixSet": "google3857",
+    "requestEncoding": "REST",
+    "resLength": "20"
+  },
+  {
+    "id": "my-capabilities-wmts",
+    "capabilitiesUrl": "WMTS capabilities url",
+    "urls": "WMTS url",
+    "optionsFromCapabilities": true,
+    "tileMatrixSet": "EU_EPSG_25832_TOPPLUS",
+    "typ": "WMTS",
+    "layers": "layer-name",
+    "legendURL": "my-legend-url"
   }
 ]
 ```
 
-Since this is the base for many functions, the service ID set in this is used to reference map material in many places of the map client.
+Since this is the base for many functions, the service id set in this is used to reference map material in many places of the map client.
+
+###### zoomOption
 
-##### <...masterportalAPI.fields>
+| fieldName | type | description |
+| - | - | - |
+| resolution | number | Size of 1 pixel on the screen converted to map units (e.g. meters) depending on the used projection (`epsg`). |
+| scale | number | Scale in meters. |
+| zoomLevel | number | Zoom level. |
 
-The `<...masterportalAPI.fields>` means that any masterportalAPI field may also be used here _directly_. The most common fields are the following ones; for more, see masterportalAPI.
+##### layer
 
 | fieldName | type | description |
 | - | - | - |
-| epsg | string | Leading coordinate system, e.g. `"EPSG:25832"`. |
-| extent | number[] | Map movement will be restricted to this rectangle. |
-| namedProjections | Array | Array of usable projections by proj4 string. |
-| options | Array | Defines all available zoomLevels. Entries define `resolution`, `scale`, and `zoomLevel`. See masterportalAPI for details. |
-| startCenter | number[] | Initial center coordinate. |
-| startResolution | number | Initial resolution; must be in options. See below. |
+| id | string | Service register id in `mapConfiguration.layerConf`. |
+| name | string | Display name in UI. |
+| type | enum["background", "mask"] | Layer handling. Backgrounds are mutually exclusive, masks ("overlays") can be stacked. |
+| maxZoom | number? | If set, layer only available (and selectable) up to this zoom level. |
+| minZoom | number? | If set, layer only available (and selectable) from this zoom level on. |
+| visibility | boolean? | Initial visibility. Defaults to `false`. |
```

Comment:
Do these have an effect in the Core (resp. masterportalapi)?

Reply:
Just looking at the docs ([services.json.md](https://bitbucket.org/geowerkstatt-hamburg/masterportal/src/dev/doc/services.json.md) and [config.json.md](https://bitbucket.org/geowerkstatt-hamburg/masterportal/src/dev/doc/config.json.md#markdown-header-themenconfighintergrundkarten)) of Masterportal, `minZoom` and `maxZoom` probably are LayerChooser specific, as `maxScale` and `minScale` is used. `visibility` seems to be usable directly though.

I've moved both to LayerChooser though, just to be safe, as I didn't find any code references in the core 71e09ee2